scripts/newinstall.sh

```diff
@@ -253,30 +354,62 @@ else:
 	fi
 fi
 
+##########	Bootstrap miniconda (optional)
+
+if true; then
+	if [[ $WITH_MINICONDA == true ]]; then
+		miniconda_path="${LSST_HOME}/miniconda${PYTHON_VERSION}-${MINICONDA_VERSION}"
+		if [[ ! -e $miniconda_path ]]; then
+			miniconda::install \
+				"$PYTHON_VERSION" \
+				"$MINICONDA_VERSION" \
+				"$miniconda_path" \
+				"$MINICONDA_BASE_URL"
+		fi
+
+		export PATH="${miniconda_path}/bin:${PATH}"
+
+		if [[ -n $CONDA_CHANNELS ]]; then
+			miniconda::config_channels "$CONDA_CHANNELS"
+		fi
+		miniconda::lsst_env "${PYTHON_VERSION}" "${LSSTSW_REF}"
+
+		CMD_SETUP_MINICONDA_SH="export PATH=\"${miniconda_path}/bin:\${PATH}\""
+		CMD_SETUP_MINICONDA_CSH="setenv PATH ${miniconda_path}/bin:\$PATH)"
+	fi
+fi
+
+# By default we use the PATH Python to bootstrap EUPS.
+# Set $PYTHON to override this or use the -P command line option.
+# $PYTHON is used to install and run EUPS and will not necessarily
```

Comment:
The `-P` option still sets `$PYTHON` but nothing uses it. Should `-P` now be setting `$EUPS_PYTHON`?